### PR TITLE
clarify flex-grow

### LINKED ISSUE #11090: Issue with "flex-grow": (unclear how remaining space is distributed, gotchas about what space is distributed)
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow

#### What information was incorrect, unhelpful, or incomplete?

1. Unclear how remaining space is distributed

    > the ratio defined by the different flex grow factors

    This should be something like

    > the ratio between the item's grow factor and the sum of all item's grow factor

2. Gotchas about what space is distributed

    The example uses items with too little content, so if someone skims they will think that it distributes the *main/total size* of the line, instead of the *remaining size*.

#### Specific section or headline?

[Description](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow#description) and [Examples](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow#examples) sections.

#### What did you expect to see?

Explained above.

#### Did you test this? If so, how?

No.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/flex-grow`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/flex-grow/index.md
* Last commit: https://github.com/mdn/content/commit/26f50bdaeb1228f458d21dbdf7dc110387b20daa
* Document last modified: 2021-08-12T23:59:09.000Z

</details>

### PR DESCRIPTION
**Pull request by estelle**
fixes https://github.com/mdn/content/issues/11090

### CODE REVIEW

File: files/en-us/web/css/flex-grow/index.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ browser-compat: css.properties.flex-grow
 
 {{CSSRef}}
 
-The **`flex-grow`** [CSS](/en-US/docs/Web/CSS) property sets the flex grow factor of a flex item's [main size](https://www.w3.org/TR/css-flexbox/#main-size).
+The **`flex-grow`** [CSS](/en-US/docs/Web/CSS) property sets the flex grow factor, which specifies how much of the flex containers remaining space should be assigned to the flex item's [main size](https://www.w3.org/TR/css-flexbox/#main-size).
```
Comment (dawei-wang) [inline files/en-us/web/css/flex-grow/index.md:10]:
```suggestion
The **`flex-grow`** [CSS](/en-US/docs/Web/CSS) property sets the flex grow factor, which specifies how much of the flex container's remaining space should be assigned to the flex item's [main size](https://www.w3.org/TR/css-flexbox/#main-size).
```

File: files/en-us/web/css/flex-grow/index.md [hunk 2]
```diff
@@ -96,6 +100,8 @@ The remaining space is the size of the flex container minus the size of all flex
 
 {{EmbedLiveSample('Setting flex item grow factor')}}
 
+When the six flex items are distributed along the container's main axis, if the sum of the main content of those flex items is lesss than the size of the containers main axis, the extra space is distributed amount the size flex items, with A, B, C, and F, each getting 12.5% of the remaining space and D and E each getting 25% of the extra space.
```
Comment (dawei-wang) [inline files/en-us/web/css/flex-grow/index.md:103]:
```suggestion
When the six flex items are distributed along the container's main axis, if the sum of the main content of those flex items is less than the size of the container's main axis, the extra space is distributed among the size flex items, with A, B, C, and F, each getting 12.5% of the remaining space and D and E each getting 25% of the extra space.
```

**Review by dawei-wang — COMMENTED** [review]
A few typos

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**